### PR TITLE
makes a bunch of lists that use typecacheof() static. doesnt find out why its overtiming at all but what the hell it helps

### PR DESCRIPTION
**Pull request by Kylerace**
## About The Pull Request
noticed typecacheof() had overtime in a round but for some reason windows explorer is hanging so ill have to get the screenshot later. i thought it was weird for it to be in the top 10 overtiming procs so i went through and i made a bunch of the lists that use it into static lists (at least for the ones in which it makes sense)

also the usual as anything's and better var names where i get annoyed enough to change them

## Why It's Good For The Game
speed? probably some.

### CODE REVIEW

File: code/modules/spells/spell_types/aimed.dm [hunk 1]
```diff
@@ -152,11 +152,12 @@
 	var/projectile_initial_spread_amount = 30
 	var/projectile_location_spread_amount = 12
 	var/datum/component/lockon_aiming/lockon_component
+	var/static/list/lockon_typecache = typecacheof(list(/mob/living))
 	ranged_clickcd_override = TRUE
 
 /obj/effect/proc_holder/spell/aimed/spell_cards/on_activation(mob/M)
-	QDEL_NULL(lockon_component)
-	lockon_component = M.AddComponent(/datum/component/lockon_aiming, 5, typecacheof(list(/mob/living)), 1, null, CALLBACK(src, .proc/on_lockon_component))
+	QDEL_NULL(lockon_component) //TODOKYLER: AAAAAAAAAAAAAAAAAAAAAHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
```
Comment (Mothblocks) [inline code/modules/spells/spell_types/aimed.dm:159]: hm

Comment (Kylerace) [reply code/modules/spells/spell_types/aimed.dm:159]: oh right the todos

**Comment by Kylerace** [comment]
fuck i knew i was forgetting something, pr will be fully unshittied tomorrow

File: code/modules/spells/spell_types/aimed.dm (hunk 1 above)
Comment (Kylerace) [reply code/modules/spells/spell_types/aimed.dm]: todo gone

File: code/modules/research/experimentor.dm [hunk 2]
```diff
@@ -35,7 +35,7 @@
 	var/list/item_reactions = list()
 	var/list/valid_items = list() //valid items for special reactions like transforming
 	var/list/critical_items_typecache //items that can cause critical reactions
-	var/banned_typecache // items that won't be produced
+	//var/banned_typecache // items that won't be produced
```
Comment (Mothblocks) [inline code/modules/research/experimentor.dm:38]: Don't leave commented out code.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/research/experimentor.dm (hunk 2 above)
Comment (Mothblocks) [reply code/modules/research/experimentor.dm:38]:
```suggestion
```

**Pull request merged**